pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1320,6 +1323,10 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			if ifaceType == "virtio" && virtioNetProhibited {
 				return fmt.Errorf("In-kernel virtio-net device emulation '/dev/vhost-net' not present")
 			} else if ifaceType == "virtio" && virtioNetMQRequested {
+				if *numQueues > multiQueueMaxQueues {
+					*numQueues = multiQueueMaxQueues
+					log.Log.Infof("Capped the number of queues to be the current maximum of tap device queues - 256")
```

Comment:
Probably using a `%v` for 256 makes sense here.

Reply:
Done.